### PR TITLE
Modrinth App instructions and more refinements

### PR DESCRIPTION
**Pull request by RaptaG**
😈

### CODE REVIEW

**Comment by RaptaG** [comment]
GH does not like renaming

File: update-instructions.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ If you added or removed some mods:
 8. Modpack will now update.
 9. Run it and check the version difference in the bottom right corner.
 
+### Modrinth App
+
+TODO
```
Comment (Madis0) [inline update-instructions.md:35]: someone can continue this

Comment (RaptaG) [reply update-instructions.md:35]: So only this is missing right?

Comment (Madis0) [reply update-instructions.md:35]: For now, yeah
Not sure what else might change before the release

File: install-instructions.md [hunk 2]
```diff
@@ -22,6 +22,20 @@ Or with the launcher already open:
 4. To run the modpack, hover on the modpack icon and click `Play`
 5. Once the Minecraft Launcher has opened, click `Play` again
 
+### [Modrinth App](https://github.com/modrinth/theseus)
+
+1. Install Java [following these instructions](https://prismlauncher.org/wiki/getting-started/installing-java/).
```
Comment (Madis0) [inline install-instructions.md:27]: Still necessary?

Comment (Madis0) [reply install-instructions.md:27]: I believe not, it seems to auto-detect as of 0.5.0

File: adding-more-mods.md [hunk 3]
```diff
@@ -25,6 +25,18 @@ Method 2:
 5. The mod will be added to the FO profile
 6. Run the game by clicking `Play`. You should now have the new mod(s) installed!
```
Comment (Madis0) [inline adding-more-mods.md:27]: note to self: update listing description numbers too (4 -> 5)

**Pull request merged**